components/Form/Form.js

```diff
@@ -60,6 +62,16 @@ export default function Form({
     setCollectionId(event.target.value);
   }
 
+  function handleLanguageChangeQuestion(event) {
+    setLanguageQuestion(event.target.value);
+    card.languageQuestion = event.target.value;
```

Comment:
In React, props should be considered immutable. Mutating props directly can lead to unpredictable behaviour in your components, as React's rendering process relies on new props or state to decide when to re-render.

1. **Use State Instead of Modifying Props**: If you need to change the values received from props, you should copy these into state in your component and manage changes there.
2. **Lift State Up**: If these changes need to be reflected at a higher level (in parent components), consider using callbacks to handle these changes in the parent component.

This is not the only place where you are mutating the props.

Reply:
fixed